build.project.props

```diff
@@ -17,3 +17,5 @@ project.admin.email = ${project.admin.username}@example.com
 # -----------------------
 profile = multisite_drupal_standard
 platform.package.version = 2.4
+
+behat.generic.package.enable = 0
```

Comment:
Disabled by default?

Reply:
Yes, I think it's best to avoid the unstable builds up until we either have fixed all issues in the behat generic package or moved it out of behat since there is no need for it. I would like to move the functionality into the "dev" module that we can use so that we can add simple checks like this one:
* https://github.com/ec-europa/digital-agenda-reference/issues/84

I'd say something like a log where you can run either a drush command or press a ui button in an admin page to generate all the urls like you created for behat, clear the log, visit all pages with curl and print the report on the page or in the cli.

In my opinion watchdog checking has little to do with user functionality testing.

Reply:
I can undertsand your point. not fully agree for sure, you know that these tests are created to fill the lack of testings in our projects, with that disabled by default we will loose a big advantage of CI.

What we can do to minimize the impact of this decision?